iOS/MMIWG2S/Resources/SlideviewImages.xcassets/Rectangle 1-2.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "Rectangle 1-2.png",
```

Comment:
I would prefer more descriptive image names. I'm not sure what this is used for. We also generally don't use file extensions in the image names.